how-to-docs/network-policies.md

```diff
@@ -28,4 +28,48 @@ spec:
 
 The port number should be the same as the one that your service is listening on.
 
-For more information, see the [Kubernetes documentation on network policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+## Controlling Egress Traffic
+
+Kubernetes v1.8 with Calico v2.6 adds support to limit egress traffic via the use of Kubernetes Network Policies.
+
+An example of a policy document blocking ALL egress traffic for a given namespace is below:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: deny-all-egress
+  namespace: <your-namespace-here>
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+```
+
+**NOTE:** The above document will also prevent DNS access for all pods in the namespace. To allow DNS egress traffic via the `kube-system` namespace, you can apply the following Network Policy document within your namespace (which takes precedence over `deny-all-egress`):
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-dns-access
+  namespace: <your-namespace-here>
+spec:
+  podSelector:
+    matchLabels: {}
+  policyTypes:
+  - Egress
+  egress:
+  - to:
+    - namespaceSelector:
```

Comment:
you won't be able to resolve anything external with this. just svc.cluster.local addresses.

Reply:
I think all DNS lookups would route via kube-dns in kube-system so it should still resolve external domains. You just won't be able to egress out to hit anything other than DNS (via kube-system)

Reply:
Have a peek here at point 5 (I followed this guide in my testing): https://docs.projectcalico.org/master/getting-started/kubernetes/tutorials/advanced-policy . It shows successful resolution of Google

Reply:
when i tried it, i wasn't able to resolve google.com

Reply:
from kash: we we're missing: 
`kubectl label namespace kube-system name=kube-system`

Reply:
so guess that needs to be rolled out across our platform too